slack/0.10.0/README.md

```diff
@@ -0,0 +1,134 @@
+# Slack Notifications via Keptn Webhooks
+
+This integration shows you how to send notifications to Slack leveraging [Keptn's Webhook Service](https://keptn.sh/docs/0.10.x/integrations/webhooks/).
```

Comment:
IIRC we have no support for "latest" link in the documentation so there might be some misconnect